spec/features/address_bar_spec.rb

```diff
@@ -321,8 +321,10 @@ def query_string
     before(:all) do
       visit '/search/map'
       wait_for_xhr
-      page.execute_script("$('#map').data('map').map.setView(L.latLng(12, -34), 5)")
-      wait_for_xhr
+      synchronize do
```

Comment:
Were you able to reproduce the failure? This shouldn't fix the failure. synchronize just runs the block repeatedly until there's no exception, and there was no exception on master.